unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Materials/BasicMaterial.cs

```diff
@@ -83,12 +83,12 @@ public override IEnumerator ApplyChanges(BaseModel newModel)
                         {
                             if ( dclTexture != null )
                             {
-                                dclTexture.DetachFrom(this);
+                                dclTexture.DetachFrom(new ComponentTextureAttachment(this));
```

Comment:
What are the upsides of using a new `ComponentTextureAttachment` instance every time?